features/aanspreekvorm.feature

```diff
@@ -0,0 +1,659 @@
+      | JV        | V           |
+      | JH        | N           |
+      | JV        | P           |
+
+  Rule: Voor het bepalen van de aanspreekvorm wordt de hoffelijkheidstitel ('titre de courtoisie') gebruikt als:
+        - het geslacht van de persoon is "V"
+        - de persoon gebruikt de naam van haar (ex-)partner (aanduiding naamgebruik is ongelijk aan "E")
+        - de (ex-)partner heeft een adellijke titel
+        - met de adellijke titel van de (ex-)partner mag een hoffelijkheidstitel ('titre de courtoisie') worden gebruikt met bijbehorend aanspreekvorm:
+          | adellijke titel | hoffelijkheidstitel | aanspreekvorm            |
+          | B               | barones             | De hoogwelgeboren vrouwe |
+          | G               | gravin              | De hooggeboren vrouwe    |
+          | H               | hertogin            | De hoogwelgeboren vrouwe |
+          | M               | markiezin           | De hoogwelgeboren vrouwe |
+          | P               | prinses             | De hoogheid              |
+
+    Abstract Scenario: persoon met geslacht "V" en aanduiding naamgebruik ongelijk aan "E", heeft partner met adellijke titel met hoffelijkheidstitel
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde    |
+      | burgerservicenummer         | 999992934 |
+      | geslachtsaanduiding (04.10) | V         |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                           | waarde        |
+      | aanduiding naamgebruik (61.10) | <naamgebruik> |
+      En de persoon heeft een 'partner' met de volgende 'naam' gegevens
+      | naam                                 | waarde                    |
+      | adellijke titel of predicaat (02.20) | <adellijke titel partner> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze.aanspreekvorm   |
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
+      | naam                          | waarde        |
+      | aanschrijfwijze.aanspreekvorm | aanspreekvorm |
+
+      Voorbeelden:
+      | naamgebruik | adellijke titel partner | aanspreekvorm            |
+      | P           | B                       | De hoogwelgeboren vrouwe |
+      | V           | G                       | De hooggeboren vrouwe    |
+      | N           | H                       | De hoogwelgeboren vrouwe |
+      | P           | M                       | De hoogwelgeboren vrouwe |
+      | V           | P                       | De hoogheid              |
+
+    Abstract Scenario: persoon met geslacht "V" en aanduiding naamgebruik ongelijk aan "E", heeft partner met adellijke titel zonder hoffelijkheidstitel of predicaat
```

Comment:
In de tabel die start in regel 41 zie je dat er rekening gehouden wordt met foutjes in de registratie m.b.t. de vastgelegde titel of predicaat door dan toch de bij het geslacht gewenste aanspreekvorm te genereren.
Wordt het uitgangspunt om foutjes op dat punt te corrigeren in de aanspreekvorm in deze gevallen losgelaten?

Dus bijv. als de partner een man is maar in de registratie staat per abuis de titel 'BS' wordt hier voor diens vrouw dus geen hoffelijkheidstitel of predicaat gegenereerd in de aanspreekvorm.